installer/README.md

```diff
@@ -5,3 +5,18 @@
   * Install the [WiX Toolset build tools](https://wixtoolset.org/releases/) in the development machine.
   * Open `powertoys.sln`, select the "Release" and "x64" configurations and build the `PowerToysSetup` project.
   * The resulting installer will be built to `PowerToysSetup\bin\Release\PowerToysSetup.msi`.
+
+## Building and installing self-signed PowerToys MSIX package
+* Make sure you've built correct `powertoys.sln` configuration
+* Add `PowerToys_TemporaryKey.pfx` (or use your own) to the [TRCA store](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/trusted-root-certification-authorities-certificate-store)
```

Comment:
Can we have a script that generates the cert instead of providing one? Some people will get this installed on their machines and that's asking for trouble.